tests/ui/associated-type-bounds/associated-item-through-where-clause.rs

```diff
@@ -1,4 +1,4 @@
-// check-pass
+//@check-pass
```

Comment:
I don't think ui-test supports this

Reply:
According to https://github.com/oli-obk/ui_test/blob/f03c5de5185f2026c65f690f74b1629505eec65b/src/parser.rs#L502 I think it does? Unless it means something different in compiletest and ui_test.